<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fileForm</class>
 <widget class="QWidget" name="fileForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
 QPushButton#pushButton {
     background-color: lightblue;
     border-width: 2px;
     border-radius: 10px;
     font: bold 14px;
     min-width: 10em;
     padding: 6px;
 }</string>
   </property>
   <property name="text">
    <string>Построить график</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>274</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Начало измерений</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Количество измерений</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEdBegin">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdCount">
      <property name="text">
       <string>100</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QLabel" name="lbName2">
      <property name="text">
       <string>Модуль 2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="lbName6">
      <property name="text">
       <string>Модуль 6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="lbName5">
      <property name="text">
       <string>Модуль 5</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lbName1">
      <property name="text">
       <string>Модуль 1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="comboBox1">
      <item>
       <property name="text">
        <string>ADC1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ADC2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccZ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox2">
      <item>
       <property name="text">
        <string>ADC1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ADC2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccZ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="lbName4">
      <property name="text">
       <string>Модуль 4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="lbName3">
      <property name="text">
       <string>Модуль 3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QComboBox" name="comboBox5">
      <item>
       <property name="text">
        <string>ADC1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ADC2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccZ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QComboBox" name="comboBox6">
      <item>
       <property name="text">
        <string>ADC1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ADC2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccZ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QComboBox" name="comboBox4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <item>
       <property name="text">
        <string>ADC1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ADC2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccZ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox3">
      <item>
       <property name="text">
        <string>ADC1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ADC2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AccZ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>T</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
